include/NAS2D/StringUtils.h

```diff
@@ -24,6 +24,9 @@ namespace NAS2D
 	std::vector<std::string> splitSkipEmpty(std::string str, char delim = ',');
 	std::pair<std::string, std::string> splitOnFirst(const std::string& str, char delim);
 	std::pair<std::string, std::string> splitOnLast(const std::string& str, char delim);
+	std::vector<std::string> splitOnUnquoted(const std::string& str, char delim = ',');
+	std::vector<std::string> splitOnUnquotedSkipEmpty(const std::string& str, char delim = ',');
```

Comment:
The `splitOnUnquotedSkipEmpty` method strikes me as almost never desirable. An empty quoted string is often used to explicitly specify a string value which is empty. Discarding such strings seems to go against the very reason for specifying them.

Reply:
It's useful for parsing files you know to have blank lines, like config files: `auto lines = splitOnUnquotedSkipEmpty(entireFileStr, '\n');`. This will output all the non-empty lines in the file.

Reply:
That's a pretty good example. That should be included in the documentation, and perhaps in a test case.